src/conn/conn_dhandle.c

```diff
@@ -818,6 +818,15 @@ __wt_conn_dhandle_discard(WT_SESSION_IMPL *session)
         goto restart;
     }
 
+    /*
```

Comment:
As discussed earlier, Check to close the metadata also and reopen it works or it has any problem?

Reply:
@kommiharibabu I tested by removing this return statement and closing the metadata cursor in `__wt_metadata_cursor_close`, but looks like the `session->meta_cursor` is not set properly and it aborts while closing the cursor.

Reply:
@raviprakashgiri29 I still feel like not closing the metadata may have problems as it contains the transaction information. Can you check for the reasons behind the failure in closing metadata and resolve it?

Reply:
@kommiharibabu I checked the code and we are already closing all the cursors (iterating cursors from `session->cursors` even metadata cursor) in `__wt_session_close_open_cursors` and that's why re-closing in `__wt_conn_dhandle_discard` caused the problem. Later the metadata is opened in any of the `__wt_metadata_search` function call.